src/latexify/codegen/function_codegen.py

```diff
@@ -252,7 +252,15 @@ def visit_FunctionDef(self, node: ast.FunctionDef) -> str:
         body_strs: list[str] = []
 
         # Assignment statements (if any): x = ...
-        for child in node.body[:-1]:
+        for i, child in enumerate(node.body[:-1]):
+            # Allow docstrings
+            if (
+                i == 0
```

Comment:
I think this condition is not necessary. In general all `Expr` with `Constant` has no side effects and can be skipped.